docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -0,0 +1,134 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I'm quite confuse - why do we keep docker-compose in docs directory?

Reply:
Besause this is an example for end-users. It's not intended to be run as regular part of development process.

Reply:
Because this file is part of the documentation package, to make it easier to release with each new documentation release. We should not link to files that are in Github or other places in the official documentation, if we describe the official installation.

> PUBLICATION
> Projects SHALL publish official releases and SHALL NOT publish unreleased materials outside the development community.

>During the process of developing software and preparing a release, various packages are made available to the development community for testing purposes. Projects MUST direct outsiders towards official releases rather than raw source repositories, nightly builds, snapshots, release candidates, or any other similar packages. The only people who are supposed to know about such developer resources are individuals actively participating in development or following the dev list and thus aware of the conditions placed on unreleased materials.

If we are going to publish this file elsewhere, we will have to have a separate publishing process for this file. As for the directory, it is the same directory as docker.rst so as not to create additional confusion. Now the entire guide is in one directory.